frame/staking/src/lib.rs

```diff
@@ -724,11 +732,12 @@ enum Releases {
 	V5_0_0, // blockable validators.
 	V6_0_0, // removal of all storage associated with offchain phragmen.
 	V7_0_0, // keep track of number of nominators / validators in map
+	V8_0_0, // Populate `SortedListProvider` with the assumption that it is pallet-bags-list
```

Comment:
I don't see where this assumption about pallet-bags-list is made ?

Reply:
I more mean to say we have only tested with pallet-bags-list and there may be assumptions that don't apply to other impls of SortedListProvider. But yeah I will remove